mypy/dmypy.py

```diff
@@ -196,14 +162,67 @@ def wait_for_server(timeout: float = 5.0) -> None:
     sys.exit("Timed out waiting for daemon to start")
 
 
+@action(status_parser)
+def do_status(args: argparse.Namespace) -> None:
+    """Print daemon status.
+
+    This verifies that it is responsive to requests.
+    """
+    status = read_status()
+    if args.verbose:
+        show_stats(status)
+    check_status(status)
+    try:
+        response = request('status', timeout=5)
+    except BadStatus:
+        raise
+    except Exception as err:
+        print("Daemon is stuck; consider %s kill" % sys.argv[0])
```

Comment:
Should we display some information about the exception, since we are catching arbitrary exceptions? An exception could be caused by a bug in dmypy.